### PR TITLE
Proposal: new properties `ml:annotationOfField` and `ml:format` especially useful for bounding boxes.

### PR DESCRIPTION
**Pull request by marcenacp**
- `ml:annotationOfField` shows the field in the ml:RecordSet that is being annotated, e.g. in this example the bounding box is the annotation for an image.
- `ml:format` is the format of the bounding box, as per [Keras bounding box format](https://keras.io/api/keras_cv/bounding_box/formats/): `XYXY`, `XYWH`, etc.

Other types (like `sc:AudioObject`) will need additional properties.

### CODE REVIEW

**Comment by marcenacp** [comment]
Offline-discussions with Omar and Pierre:

- `format` should go in `extract.format` (new property).
- The relation object->label is represented by https://schema.org/additionalProperty on the object (here the image): `"annotatedBy": "label"`
